lib/version_utils.pm

```diff
@@ -785,7 +785,7 @@ sub verify_os_version {
     my ($version, $os_release_file) = @_;
     $version //= get_var("VERSION");
     $os_release_file //= '/etc/os-release';
-    return script_output("grep VERSION= $os_release_file | grep $version");
+    return script_output("grep 'VERSION=\\|VARIANT=' $os_release_file | grep $version");
```

Comment:
you are missing to document the function for whatever reason you add VARIANT here.